README.md

```diff
@@ -71,6 +71,17 @@ If you wish to run tests locally, you will need to ensure that `redis-server` is
 
 Once Redis is installed, run our entire test suite with `yarn test`.
 
+### Windows 10 Installation on Windows Subsystem for Linux 2
+
+1. First [Install WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) choosing Ubuntu or distribution of choice.
+2. [Install NVM](https://simplernerd.com/install-node-npm-wsl2/) to allow for different versions of Node.  This will also install NPM.
+3. Install version 14.3.0 by running `nvm install 14.3.0`.
+4. If you install multiple versions run `nvm use 14.3.0` to switch to the version used by Upswyng.
+5. Install yarn globally using npm with `sudo npm install -g yarn`.
```

Comment:
```suggestion
5. [Install yarn globally](https://classic.yarnpkg.com/en/docs/install)
```
That way there's less for us to manage in the future if `yarn` installation changes.